published_topics:
  flir_image_topic: /flir_lepton/image
  flir_temper_topic: /flir_lepton/temperatures
  flir_batch_topic: /flir_lepton/batch
flir_urdf:
  camera_optical_frame: /flir_lepton_optical_frame
thermal_image:
  height: 60
  width: 80
iface:
  device_port: "/dev/spidev0.0"
  bits: 8
  speed: 16000000
  delay: 0
  packet_size: 164
  packets_per_frame: 60
frame_rate: 20
dataset_file_name: "dataset_spline"
